src/components/MetricsOptions/MetricsSettingsDropdown.tsx

```diff
@@ -136,13 +193,36 @@ export class MetricsSettingsDropdown extends React.Component<Props, State> {
         {/* TODO: Remove the class="pf-c-dropdown__menu-item" attribute which is fixing a sizing issue in PF.
          * https://github.com/patternfly/patternfly-react/issues/3156 */}
         <div style={{ paddingLeft: '10px', backgroundColor: PFColors.White }} className="pf-c-dropdown__menu-item">
+          {hasLabels && this.renderBulkSelector()}
           {hasLabels && this.renderLabelOptions()}
           {hasHistograms && this.renderHistogramOptions()}
         </div>
       </Dropdown>
     );
   }
 
+  renderBulkSelector(): JSX.Element {
+    return (
+      <div>
+        <DropdownToggleCheckbox
+          id="bulk-select-id"
+          key="bulk-select-key"
+          aria-label="Select all metrics and labels"
```

Comment:
What about "Select all metric/label filters" ? "Select all metrics and labels" sounds a little bit odd, as user can't select metrics (charts) but just the filtering.

Reply:
Yes, I agree, sounds better!.